components/modal/themes/spectrum-two.css

```diff
@@ -21,7 +21,7 @@
 		--spectrum-modal-max-height: 90vh;
 		--spectrum-modal-max-width: 90%;
 
-		--spectrum-modal-background-color: var(--spectrum-gray-75);
```

Comment:
If we're changing this value for Spectrum 2, shouldn't this be set back to the existing S1 value for S1/Legacy in `spectrum.css`?

I was also wondering if the design team has a new token that should be used for this background color, rather than us creating a custom token that would only be used temporarily. That might be worth confirming with the designers in `#spectrum_web_alliance`. Looking at the S2 Figma for the dialogs, it seems like it could be `background-layer-2-color`, but that has different values than I'm seeing here listed for both S1 and S2:

<img width="316" alt="Screenshot 2024-09-05 at 10 45 29 AM" src="https://github.com/user-attachments/assets/8121208a-3c4d-4526-a643-8ad7055a9b6d">

Reply:
Thanks @jawinn The new token is `--spectrum-background-layer-2-color`

Reply:
Looks good. Thanks for confirming that with the design team.